ocs_ci/utility/lvmo_utils.py

```diff
@@ -0,0 +1,145 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import Pod, get_pods_having_label
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import LVMOHealthException
+
+log = logging.getLogger(__name__)
+
+
+def get_lvm_vg_manager_pod(
```

Comment:
https://github.com/red-hat-storage/ocs-ci/blob/e9f05c0795aa70247a67093a9e918601f207f996/ocs_ci/ocs/resources/pod.py#L2582
This function can replace line 12-85. It checks whether the PODs in a given namespace are running. Even send it with defaults and it will work.

Reply:
i will use this function, but i would like to keep some of the functions of get_specifictype_pod, same as we already have [here](https://github.com/red-hat-storage/ocs-ci/blob/5192d7601f9e8bc709c70b3dedba09afa97d31c3/ocs_ci/ocs/resources/pod.py#L693)

Reply:
Can you think of a reason to get one of the pods?

Reply:
for future test, yes. same as we do in other ODF scenarios

Reply:
move it to pod.py, same as other get_specific_pod functions